### PR TITLE
Feature/ap runner

### PR DESCRIPTION
**Pull request by aplasket**
## High level what you did: 
- changed file i/o to runner, added files to spec helper, and added class method 

### Details:
- added a self.from_csv(csv) method to stat tracker class
- add runner and stat tracker to spec_helper
- updated file i/o to be named runner.rb
- removed pry from runner file

## Overall:
- [ ] Tests are written for all methods added
- [x] Rspec running clean
- [ ] Bugs, need help, etc.

### CODE REVIEW

File: lib/stat_tracker.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+require "csv"
+
+class StatTracker
```
Comment (goldenll) [inline lib/stat_tracker.rb:3]: this runs without errors

**Review by goldenll — COMMENTED** [review]
Looks good to me, we saw the files run without errors during our group meeting. Thanks for catching the name change!

File: lib/runner.rb [hunk 2]
```diff
@@ -12,4 +12,5 @@
   game_teams: game_teams_path
 }
 
-stat_tracker = StatTracker.from_csv(locations)
\ No newline at end of file
+require 'pry'; binding.pry
```
Comment (MWMJohnson) [inline lib/runner.rb]: Do we need to remove the pry before merging?

Comment (aplasket) [reply lib/runner.rb]: done!

Comment (MWMJohnson) [reply lib/runner.rb]: Thanks Ashley!

**Pull request merged**